yuareg.el

```diff
@@ -1760,6 +1780,8 @@ Return values can be
    ((and (eq kind :before) (yuareg-smie--monadic-rule token)))
    ((and (equal token "and") (smie-rule-parent-p "type"))
     0)
+   ((and (equal token "[") (smie-rule-next-p "@@"))
```

Comment:
Now, a syntax begin from `[ @@` will be treated as [item-attribute](https://caml.inria.fr/pub/docs/manual-ocaml/extn.html#sec260)